less/navigation.less

```diff
@@ -1 +1,25 @@
-// Navigation code here
\ No newline at end of file
+// Navigation code here
+.container home {
+    display: flex;
+}
+.space-top {
+    
+    height: 80px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    background: #e6e3e3;
+}
+.top {
+    font-size: 30px;
+    display: flex;
+    justify-content: center;
+    
+}
+nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    width: 325px;
+}
```

Comment:
you can use percentage in the width to insure that it stays responsive across all viewport widths